Makefile

```diff
@@ -481,7 +481,7 @@ endif
 	rm -fr $(BUILDROOT)/julia-$(JULIA_COMMIT)
 
 # this target does not accept BUILDROOT
-light-source-dist.tmp: $(JULIAHOME)/doc/_build/html
+light-source-dist.tmp: $(BUILDROOT)/doc/_build/html/en/index.html
```

Comment:
The `tar` command expects a specific repo layout, specifically, one where the generated files (like `version_git.jl`) overlay the `base/` folder. We don't support out-of-tree builds for the source-dist target since these generated files need to live in-tree for the tarball.